### DISCUSSION TITLE
Get telcos on board

### DISCUSSION THREAD

**Discussion by Oskar-van-Deventer** [trustoverip/trust-spanning-protocol#16] [General]
As already mentioned in [my introduction](https://github.com/trustoverip/trust-spanning-protocol/discussions/4#discussioncomment-4904856), it may be useful to get Telco's on board. Telcos are trusted parties in the communications domain, which arguable includes ToIP-layer-2 peer-to-peer communication. According to a recent article: "[Telcos lead on consumer trust when it comes to data protection and privacy](https://www.oliverwyman.com/media-center/2022/feb/telcos-lead-on-consumer-trust-when-it-comes-to-data-protection-and-privacy.html)".

Telco's could offer services related to the function of [Intermediary Systems](https://github.com/trustoverip/TechArch/blob/main/spec.md#9-intermediary-systems) and [Supporting Systems](https://github.com/trustoverip/TechArch/blob/main/spec.md#10-supporting-systems).

It would be good to write our TSP specification with potential adoption by IETF, 3GPP, ITU-T or 3GPP in mind. Especially 3GPP may be promising, as privacy may be one of the selling points of the upcoming 6G release of 3GPP standards. One suggestion would be to add a very simple role model that distinguishes "trust-services end user" and "trust-services provider", and specify use cases and requirements in trust-services terms.

Oskar

**Comment by guru-barbes** [comment]
Don't know if this helps, but here's a recent joint venture doc between DEUTSCHE TELEKOM / ORANGE / TELEFONICA / VODAFONE

"The JV will offer a privacy-led, digital identification solution to support the digital marketing and advertising activities of brands and publishers. Subject to explicit user consent provided to a brand or publisher (on an opt-in basis only), the JV will generate a secure, pseudonymized token derived from a hashed/encrypted pseudonymous internal identity linked to a user’s network subscription which will be provided by participating network operators. This token will allow the brand/publisher concerned to recognize a user without revealing any directly identifiable personal data and thereby enable them to optimize the delivery of online display advertising and perform site/app optimization. Users will have access to a user-friendly privacy portal. They can review which brands and publishers they have given consent to, and withdraw their consent."

[M.10815 - DEUTSCHE TELEKOM : ORANGE : TELEFONICA : VODAFONE : JV.pdf](https://github.com/trustoverip/trust-spanning-protocol/files/10722634/M.10815.-.DEUTSCHE.TELEKOM.ORANGE.TELEFONICA.VODAFONE.JV.pdf)

**Reply by talltree** [reply]
@Oskar-van-Deventer I definitely agree that telcos are a strong adoption target. Question: given the very general roles (really, system types) defined in the [ToIP Technology Architecture Specification V1.0](https://github.com/trustoverip/TechArch/blob/main/spec.md), where would you map the role of "trust service provider"?

**Reply by Oskar-van-Deventer** [reply]
@talltree Drummond, I do not understand your question. The architecture spec is a technical spec that specifies technical entities, like "Endpoint System", "Intermediary System" and "Supporting System". Business roles, like "Trust Service Provider" belong in a use-cases+requirements document. I am not aware of a ToIP document that defines this particular role.

**Reply by talltree** [reply]
@Oskar-van-Deventer You actually answered my question, i.e., I didn't know whether you were suggesting that the role of "trust-service provider" was missing from the architecture spec. From my perspective, the purpose of the TSP layer is universal, i.e., it applies to 100% of the roles of anyone using the ToIP stack for any trust task.

So the role of trust-service provider would apply to a specific group of the trust tasks at ToIP Layer 3. In fact, as we begin to look at the architecture of trust task protocols at Layer 3, looking at the set of trust tasks that a trust-service provider will need to accomplish (for example, under the [EU Architecture Reference Framework](https://digital-strategy.ec.europa.eu/en/library/european-digital-identity-wallet-architecture-and-reference-framework)) might be a good way to analyze those tasks.

**Reply by Oskar-van-Deventer** [reply]
@talltree  Drummond, also ToIP layer 2 has "roles", where each role may implement one or more of the technical entities from the layer-2 part of the architecture spec. What worries me, is that we don't seem to have a description of roles, use cases and requirements focussed on ToIP layer 2. Affinity with telcos may benefit from an informational description of their potential role at ToIP layer 2. My use of terminology may have been awkward in my original post. Maybe better would be to talk about a "ToIP-layer-2 user" role and a "ToIP-layer-2 service-provider" role. Telco's may recognize themselves in a "ToIP-layer-2 service-provider" role.

**Comment by Oskar-van-Deventer** [comment]
[ToIP TSP - Should 6G be trust spanning - Oskar van Deventer.pptx](https://github.com/trustoverip/trust-spanning-protocol/files/11677776/ToIP.TSP.-.Should.6G.be.trust.spanning.-.Oskar.van.Deventer.pptx)
My presentation "Should 6G be Trust Spanning", for the ToIP TSP call of 7 June 2023.

**Comment by wenjing** [comment]
Thanks @Oskar-van-Deventer for bringing this important topic to the TF. I'll summarize the two points that I made in the June 7 meeting here for reference:
- I really like your call for informational service models which can map concepts we chose to abstract (for good reasons) to practical deployments - telcos being one of them. This is an absolutely required step for ensuring the soundness of the overall approach and practicality for adoption in large scale. In addition, privacy is a complex - cross layer boundary - problem, that must be solved with the full view of a deployment, not in isolation.
- One good opening case of collaboration with the telcos may be in what I call the coming "Voice authentication" crisis (which is more immediate than 6g). With the new advances in Generative AI, voice spoofing (AI modeled human speech emulating a real person) technique is now widely available, and if combined with phone number spoofing, will create an immediate crisis in numerous services where we implicitly rely on "voice recognition" as part of authentication process. There is an immediate need to solve this identity problem. If we can collaborate with telcos to solve this problem with ToIP architecture, the telcos can extend the same solution/architecture to 5g or 6g in general as a telco provided service to any apps.
